install/system-time.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Install tzupdate if not present
+yay -S --noconfirm --needed tzupdate
+
+# Ensure the current user is in the wheel group
+if ! groups $USER | grep -qw wheel; then
+  echo "Adding $USER to wheel group (admin privileges required)..."
+  sudo usermod -aG wheel $USER
+  echo -e "\e[32m$USER added to wheel group. You may need to log out and back in for this to take effect.\e[0m"
+fi
+
+# Create sudoers file for tzupdate and timedatectl set-timezone
+SUDOERS_FILE="/etc/sudoers.d/omarchy-tzupdate"
+SUDOERS_CONTENT="%wheel ALL=(root) NOPASSWD: /usr/bin/tzupdate, /usr/bin/timedatectl set-timezone *"
```

Comment:
@dhh  @ryanrhughes  Thoughts on this?

- This feels a little dirty to me, I know this is somewhat how Ubuntu is pulling this off however.
- We could just have the user enter their password?
- We could switch to a different package, `timedatectl` built in can change without sudo password